packages/batch-tx-handler/package.json

```diff
@@ -35,7 +35,7 @@
     "ethereumjs-util": "^7.1.5",
     "ethers": "^5.7.0",
     "js-base64": "^3.7.5",
-    "node-fetch": "^2.7.0",
+    "@types/node-fetch": "^2.6.9",
```

Comment:
you need to install `"@types/node-fetch": "^2.6.9",` by running `yarn add --dev @types/node-fetch` and keep your `node-fetch`

the `@types` package is only for type definitions, not the `node-fetch` package itself.